Automattic/jetpack
# Connection js-package: Add some debug statements

Adds some debug statements to a couple of components from the connection js-package

## Proposed changes:

* Adds some debug statements to useProductCheckoutWorkflow connection component.
* Adds some debug statements to ConnectScreenRequiredPlanVisual connection component.

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?
No

## Testing instructions:
* On a disconnected site with a plugin like Jetpack backup
* Visit the Backup page
* open the console
* Write `localStorage.debug='*'`
* Refresh the page
* Expect to see debug messaged logged into the console coming from the connection package. e.g.
  ```
  common.js:113 jetpack:connection:useProductCheckoutWorkflow productSlug is jetpack_backup_t1_yearly +3ms
  common.js:113 jetpack:connection:useProductCheckoutWorkflow redirectUrl is admin.php?page=jetpack-backup +0ms
  common.js:113 jetpack:connection:useProductCheckoutWorkflow siteSuffix is hello-dolly.jurassic.ninja +0ms
   ```
